.buildbot.sh

```diff
@@ -117,6 +135,13 @@ for _ in $(seq 10); do
     RUST_TEST_SHUFFLE=1 cargo test
     YKD_NEW_CODEGEN=1 RUST_TEST_SHUFFLE=1 cargo test
 done
+
+# test yklua/hwt in debug mode.
+OLDPATH=${PATH}
```

Comment:
This is truly horrible :p Can we not just set `PATH=...` in the line below, so it's local to `test_yklua`, just as with the other env vars? Pushing/popping `$PATH` wouldn't be necessary then.

Reply:
I don't know why I didn't think of that.

Reply:
Try https://github.com/ykjit/yk/pull/1286/commits/706ce13c4f7cf04ff2c2be9886ff0ed5c110cad4

Reply:
Everything is obvious in retrospect!